src/System.Runtime.Caching/src/Configurations.props

```diff
@@ -1,13 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <PackageConfigurations>
-      netstandard;
-      netcoreapp2.0-Windows_NT;
-    </PackageConfigurations>
     <BuildConfigurations>
-      $(PackageConfigurations);
-      netcoreapp-Windows_NT;
+      netstandard-Windows_NT;
```

Comment:
We generally avoid making netstandard assets OS specific, as netstandard isn't a concrete platform it is a means of sharing and can be supported everywhere. I suggest instead you keep what we had before and just add a netcoreapp2.0-Unix configuration.

Reply:
Makes sense. Thanks you very much for the feedback.